workers/tasks/export/html.js

```diff
@@ -0,0 +1,250 @@
+				};
+			}
+			if (node.type === 'footnote') {
+				return {
+					href: `#${footnotePrefix}-${count}`,
+					id: `${footnotePrefix}-${count}-return`,
+				};
+			}
+			return {};
+		},
+		['citation', 'footnote'],
+		nodes,
+	);
+
+const blankIframes = (nodes) => addAttrsToNodes({ url: 'about:blank' }, ['iframe'], nodes);
+
+const renderFrontMatterForPandoc = ({ updatedDateString, publishedDateString, doi }) => {
+	const showUpdatedDate = updatedDateString && updatedDateString !== publishedDateString;
+	return (
+		<React.Fragment>
+			{showUpdatedDate && (
+				<div>
+					<strong>Updated on:</strong> {updatedDateString}
+				</div>
+			)}
+			{doi && (
+				<div>
+					<strong>DOI:</strong> {doi}
+				</div>
+			)}
+		</React.Fragment>
+	);
+};
+
+const renderFrontMatterForHtml = ({
+	updatedDateString,
+	publishedDateString,
+	primaryCollectionTitle,
+	doi,
+	title,
+	communityTitle,
+	attributions,
+}) => {
+	const showUpdatedDate = updatedDateString && updatedDateString !== publishedDateString;
+	const affiliations = [
+		...new Set(attributions.map((attr) => attr.affiliation).filter((x) => x)),
+	];
+	return (
+		<section className="cover">
+			<h3>
+				<span className="community">{communityTitle}</span>
+				{primaryCollectionTitle && (
+					<span className="collection">
+						{bullet}
+						{primaryCollectionTitle}
+					</span>
+				)}
+			</h3>
+			<h1 className="title">{title}</h1>
+			{attributions.length > 0 && (
+				<div className="byline">
+					<h3>
+						{attributions.map((attr, index) => {
+							const {
+								user: { fullName },
+								affiliation,
+							} = attr;
+							const affiliationNumber =
+								affiliation && affiliations.includes(affiliation)
+									? 1 + affiliations.indexOf(affiliation)
+									: null;
+							return (
+								<span className="name" key={index}>
+									{fullName}
+									{affiliationNumber && <sup>{affiliationNumber}</sup>}
+									{index < attributions.length - 1 && ', '}
+								</span>
+							);
+						})}
+					</h3>
+					{affiliations.length > 0 && (
+						<h5>
+							{affiliations.map((aff, index) => (
+								<span className="affiliation" key={index}>
+									{<sup>{index + 1}</sup>}
+									{aff}
+									{index < affiliations.length - 1 && ', '}
+								</span>
+							))}
+						</h5>
+					)}
+				</div>
+			)}
+			<div className="details">
+				{publishedDateString && (
+					<div>
+						<strong>Published on: </strong> {publishedDateString}
+					</div>
+				)}
+				{showUpdatedDate && (
+					<div>
+						<strong>Updated on:</strong> {updatedDateString}
+					</div>
+				)}
+				{doi && (
+					<div>
+						<strong>DOI:</strong> {doi}
+					</div>
+				)}
+			</div>
+		</section>
+	);
+};
+
+export const createStaticHtml = async (
+	{ prosemirrorDoc, pubMetadata, citations, footnotes },
+	targetPandoc,
+	targetPaged,
+) => {
+	const { title } = pubMetadata;
+
+	const renderableNodes = [
+		filterNonExportableNodes,
```

Comment:
Something here seems to not be catching. I'm getting an `Unknown node type: discussion` error when trying to export [Resisting Reduction](https://jods.duqduq.org/pub/resisting-reduction).

![Screen Shot 2019-11-19 at 10 49 19 AM](https://user-images.githubusercontent.com/1000455/69140451-7026b080-0aba-11ea-9b1b-022a2c04d2ff.png)

Reply:
Fixed! I think this is happening in prod, too :|